.editorconfig

```diff
@@ -1,53 +0,0 @@
-# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
Editor config is something we use in all our projects, to help keep consistency with things like line spacing and indentation. Let's not delete it here, and I'd encourage everyone to [configure your editor](http://EditorConfig.org) to use it. It helps save time in code review and keeps things consistent 👍

Reply:
This is good to know 👍, I was rolling back some of the linting work that Craig had done previously and didn't quite understand what this file was doing. I added it back in.